dallinger/__init__.py

```diff
@@ -7,12 +7,18 @@
     networks,
     processes,
     transformations,
+    experiment,
     experiments,
     heroku
 )
 
+import logging
+from logging import NullHandler
 from localconfig import config
 config.read("config.txt")
+logger = logging.getLogger(__name__)
+logger.addHandler(NullHandler())
```

Comment:
To avoid spew about no log handler being registered. I was following the logging in a library best practices here: http://docs.python-guide.org/en/latest/writing/logging/

The command line script is the "Application" so I real log handler setup there.